spec/classes/software_vcsscm_git_spec.rb

```diff
@@ -15,6 +15,65 @@
           it { is_expected.to contain_package('git') }
         end
       end
+
+      context 'with booleans for gitconfig and gitignore'
```

Comment:
the `do` keyword is missing from this block.

Reply:
:+1:

_The GitHub reactions feature seems to be broken. I have to write a comment for "thumbs up"._  :open_mouth: